README.adoc

```diff
@@ -67,40 +74,42 @@ The following is a basic example of using the action:
       environment: test_env
       steps:
         - name: Register deployed artifact
-          uses: cloudbees-io/register-deployed-artifact@v1
+          uses: cloudbees-io/register-deployed-artifact@v2
           with:
-            artifact_id: "00000000-0000-0000-0000-000000000001"
-            target-environment: "test_env"
+            artifact_id: 1234abcd-56ef-ab78-9012-cdef78903456
+            target-environment: test_env
```

Comment:
It is confusing to have both `job.environment` and `target-environment` in a basic example. Lets just specify one of them here.